[2024-12-06T14:00:01.067+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-06T14:00:01.080+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: get_inaturalist_data_atropelamento.get_all_data scheduled__2024-12-06T13:30:00+00:00 [queued]>
[2024-12-06T14:00:01.088+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: get_inaturalist_data_atropelamento.get_all_data scheduled__2024-12-06T13:30:00+00:00 [queued]>
[2024-12-06T14:00:01.088+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 4
[2024-12-06T14:00:01.101+0000] {taskinstance.py:2888} INFO - Executing <Task(_PythonDecoratedOperator): get_all_data> on 2024-12-06 13:30:00+00:00
[2024-12-06T14:00:01.110+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=1581) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-12-06T14:00:01.112+0000] {standard_task_runner.py:72} INFO - Started process 1586 to run task
[2024-12-06T14:00:01.112+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'get_inaturalist_data_atropelamento', 'get_all_data', 'scheduled__2024-12-06T13:30:00+00:00', '--job-id', '2580', '--raw', '--subdir', 'DAGS_FOLDER/inaturalist_dag.py', '--cfg-path', '/tmp/tmpjbx64z84']
[2024-12-06T14:00:01.114+0000] {standard_task_runner.py:105} INFO - Job 2580: Subtask get_all_data
[2024-12-06T14:00:01.157+0000] {task_command.py:467} INFO - Running <TaskInstance: get_inaturalist_data_atropelamento.get_all_data scheduled__2024-12-06T13:30:00+00:00 [running]> on host 4d2800f2c921
[2024-12-06T14:00:01.240+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='get_inaturalist_data_atropelamento' AIRFLOW_CTX_TASK_ID='get_all_data' AIRFLOW_CTX_EXECUTION_DATE='2024-12-06T13:30:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-12-06T13:30:00+00:00'
[2024-12-06T14:00:01.243+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-12-06T14:00:01.245+0000] {inaturalist_dag.py:153} INFO - Getting data from keyword: atropelamento
[2024-12-06T14:00:01.250+0000] {inaturalist_dag.py:45} INFO - Maximum id 253150365 for the request
[2024-12-06T14:00:02.774+0000] {inaturalist_dag.py:61} INFO - Got 0 results
[2024-12-06T14:00:02.774+0000] {inaturalist_dag.py:160} INFO - No more observations found, stopping.
[2024-12-06T14:00:02.780+0000] {crypto.py:82} WARNING - empty cryptography key - values will not be stored encrypted.
[2024-12-06T14:00:02.784+0000] {python.py:240} INFO - Done. Returned value was: []
[2024-12-06T14:00:02.805+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-12-06T14:00:02.805+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=get_inaturalist_data_atropelamento, task_id=get_all_data, run_id=scheduled__2024-12-06T13:30:00+00:00, execution_date=20241206T133000, start_date=20241206T140001, end_date=20241206T140002
[2024-12-06T14:00:02.851+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-12-06T14:00:02.873+0000] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-12-06T14:00:02.875+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
